python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -198,6 +197,16 @@ def load(self) -> bool:
                 raise ValueError(
                     f"Unsupported task {self.task}. Please check the supported `task` option."
                 )
+            if not self.tokenizer.pad_token:
```

Comment:
Seems like this is moved to only for the predictor case, for the transformer mode do we still need to apply the padding ?

Reply:
We won't have access to the model to update the vocabulary size. So, Even if we add the pad token we will get an `Index out of range ` error.